frontend/src/scenes/dashboard/DashboardItem.js

```diff
@@ -125,156 +126,179 @@ export function DashboardItem({
             data-attr={'dashboard-item-' + index}
         >
             <div className={`dashboard-item-container ${className}`}>
-                <div className="dashboard-item-header">
+                <div className="dashboard-item-header" style={{ cursor: inSharedMode ? 'auto' : 'move' }}>
```

Comment:
Might be nicer to keep all the style rules together in the css and control via classnames.